orchestra/cmds/configure.py

```diff
@@ -17,17 +17,20 @@ def install_subcommand(sub_argparser):
 
 
 def handle_configure(args):
-    config = Configuration(fallback_to_build=args.fallback_build,
-                           force_from_source=args.from_source,
-                           use_config_cache=args.config_cache,
-                           )
+    config = Configuration(
+        fallback_to_build=args.fallback_build,
+        force_from_source=args.from_source,
+        use_config_cache=args.config_cache,
+    )
     build = config.get_build(args.component)
 
     if not build:
         suggested_component_name = config.get_suggested_component_name(args.component)
         logger.error(f"Component {args.component} not found! Did you mean {suggested_component_name}?")
         return 1
 
+    args.no_merge = False
+    args.keep_tmproot = False
```

Comment:
Can you explain?

Reply:
`ConfigureAction` can trigger other `InstallAction`s, which need a value for those cmdline arguments.
I thought about adding them to `orc configure`, but it does not make a lot of sense to have them: since they are global, specifying them would "always" result in the `ConfigureAction` to fail.
The rationale is, since the InstallAction is triggered because the configure script needs to install a dependency, not merging it into the root would break it.
The `--keep-tmproot` option is instead meant for debugging builds that fail (or fail to place files in the expected location, etc), but in that case it makes more sense to debug the failing dependency by using `orc install --keep-tmproot` directly, not by invoking `orc configure` on something which requires the failing component.

Reply:
Mmmh, the hack suggest that we either need to:

1. Stop forwarding `args` to `Executor` and make all the things he needs explicit, or...
2. Have one and only one user of `Executor` (i.e., the `install` command).

I'd go for option 1).

Reply:
Done